Blog/urls.py

```diff
@@ -12,4 +12,3 @@
     # url for inserting into the db (lookup on views.py)
     # path('insert/', views.insert_to_db),
 ]
```

Comment:
I have no clue how this happened, the line is still there locally.

Reply:
You could remove this unused code in a commit of its own.

Reply:
Sorry there is no need to remove it -> this code is part of the script which adds many entities to the database. I should consider to read my own comments.